README.md

```diff
@@ -1,7 +1,69 @@
+10. Using the credentials you just created, login. *If you find yourself being redirected to the login page on successful credentials, change the browser you are using.* 
+11. Navigate to [Clients](https://dev.squarelet.com/admin/oidc_provider/client/)
+12. Create a client called `MuckRock Dev`
+13. Make sure the fields have the following values:
+
+|Field| Value |
+|--|--|
+| Owner | blank/your user account |
+|Client Type|Confidential|
+|Response Types|code (Authorization Code Flow)|
+|Redirect URIs (on separate lines)|http://api.dev.documentcloud.org/accounts/complete/squarelet http://minio.documentcloud.org/accounts/complete/squarelet|
+|JWT Algorithm|RS256|
+|Require Consent?|Unchecked|
+|Reuse Consent|Checked|
+|Client ID|This will be filled in automatically upon saving|
+|Client SECRET|This will be filled in automatically upon saving|
+|Scopes (on separate lines)|read_user read_organization read_auth_token|
+|Post Logout Redirect URIs|http://dev.documentcloud.org|
+|Webhook URL (To make this field appear, Add a client profile)|http://api.dev.documentcloud.org/squarelet/webhook/|
+
+14. Make sure in your **Squarelet project** `.envs/.local/.django` file, there exist values for: `STRIPE_PUB_KEYS`, `STRIPE_SECRET_KEYS`, and set `STRIPE_WEBHOOK_SECRETS=None`. Multiple values for any of the above should be comma delimited.
+
+- You must always fully `docker-compose down` or Ctrl-C each time you change a `.django` file of a docker-compose session for it to take effect.
+
+- Avoid changing Squarelet's `.django` file frequently to prevent Docker network problems from `docker-compose down`.
+
+Be sure to stop (if needed) both the docker compose sessions DocumentCloud (Ctrl-C, or `docker-compose down`) and Squarelet (`docker-compose down` in Squarelet folder). Then run the Squarelet session using `inv up` in the squarelet folder. **Finally, run `docker-compose up` in this DocumentCloud folder to begin using the new dotfiles.**
+-   This will build and start all of the DocumentCloud docker images using docker-compose. It will attach to the Squarelet network which must be already running. You can connect to Squarelet nginx on port 80 and it will serve the appropriate dependent http service, such as DocumentCloud, based on domain as a virtual host. The `local.yml` configuration file has the docker-compose details.
+- If you do `docker-compose down` on Squarelet when none of the other dependent docker-compose sessions (such as DocumentCloud) are running, `docker-compose down` will delete the Squarelet network. You will have to explicitly bring the whole squarelet docker-compose session back up to recreate it and nginx before being able to start a dependent docker-compose session (such as DocumentCloud).
+- Using `docker-compose up -d` rather than `docker-compose up` will make a daemon for DocumentCloud as Squarelet defaults to.
```

Comment:
I think we decided to use `docker-compose up` everywhere with a possible note that you can use `-d` to daemonize.  I think that note could also be moved to the Squarelet docker section, as that is the common point of entry now.